frontend/src/home/components/NextStep.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component, PropTypes } from "react";
+
+import SidebarSection from "./SidebarSection.jsx";
+
+export default class NextStep extends Component {
+    constructor(props, context) {
+        super(props, context);
+        this.state = {
+            next: null
+        };
+    }
+
+    async componentWillMount() {
+        let response = await fetch("/api/setup/admin_checklist", { credentials: 'same-origin' });
```

Comment:
I think it's worth looking at the way this component is managed and trying to use a more redux friendly approach here which is consistent with the way we've been building things lately.  

My expectation is that componentWillMount() would simply trigger a redux action such as `fetchChecklist()` which would then be responsible for pulling the data from the api and adding it to the redux store.  This keeps the api code in the redux actions and allows the react components to avoid state.